app/main/__init__.py

```diff
@@ -1,6 +1,20 @@
-from flask import Blueprint
+from flask import Blueprint, flash, Markup, current_app
+from flask_login import login_required, current_user
+import flask_featureflags
 
 main = Blueprint('main', __name__)
+direct_award = Blueprint('direct_award', __name__, url_prefix='/buyers/direct-award')
+
+LOGIN_REQUIRED_MESSAGE = Markup("""You must be logged in to view this page.""")
```

Comment:
The equivalent message in the briefs app is "You must log in with a buyer account to see this page." and it would probably be good if these matched. Ideally they'd be shared - so I think we should potentially move `require_login` login into dm-utils. Somehow. No idea how we can share these `before_request` things between apps, but it seems like a valid need (cc @allait).

Reply:
Let's add this as tech debt because it will need to be centralised and applied across repositories.